src/integration/java/uk/nhs/prm/repo/ehrtransferservice/repo_incoming/EhrRequestTest.java

```diff
@@ -41,23 +46,33 @@ class EhrRequestTest {
     private TransferTrackerDbUtility transferTrackerDbUtility;
 
     @Autowired
-    private QueueUtility queueUtility;
+    private SQSQueueUtility SQSQueueUtility;
```

Comment:
Can we please change this to camel case?

```suggestion
    private SQSQueueUtility sqsQueueUtility;
```